app/models/mentee.rb

```diff
@@ -0,0 +1,3 @@
+class Mentee < ApplicationRecord
+  belongs_to :user
+end
```

Comment:
I am confused by what a mentee is.  Is the user the mentee or the mentor?  You could name the associate for its meaning, not just the table referenced.  Check out the :foriegn_key option when defining the relationship. I am just having a hard time understanding the relationship.